documentation/src/docs/basic-reader-and-writer.md

```diff
@@ -263,130 +224,102 @@ public void run() {
 
-So at this point in the code, we have the Scope and Stream set up, we have the
-ReaderGroup created and now we need to create a Reader and start reading Events.
+At this point, we have the Scope and Stream is set up and the `ReaderGroup` object created. Next, we need to create a Reader and start reading Events.
 
 ## Reading Event using an EventStreamReader
 
-Lines 17-36 show an example of setting up an EventStreamReader and reading
-Events using that EventStreamReader.
-
-First, we create a ClientFactory on line 17, in the same way we did it in the
-HelloWorldWriter app.  
-
-Then we use the ClientFactory to create an EventStreamReader object.  There are
-four things the developer needs to create a Reader: a name for the reader, the
-readerGroup it should be part of, the type of object expected on the Stream, the
-serializer to use to convert from the bytes stored in Pravega into the Event
-objects and a ReaderConfig.  Lines 18-21 show the creation of an
-EventStreamReader.  The name of the Reader can be any valid Pravega name
-(numbers and letters).  Of course, the name of the reader is namespaced within
-the Scope.  We talked about the creation of the ReaderGroup in the previous
-section.  Just like with the EventStreamWriter, EventStreamReader uses Java
-generic types to allow a developer to specify a type safe Reader.  In our case,
+First, we create a `ClientFactory` object, the same way we did it in the
+`HelloWorldWriter` application. Then we use the `ClientFactory` to create an `EventStreamReader` object. The following are the four parameters to create a Reader:
+
+ - Name for the Reader.
+ - Reader Group it should be part of.
+ - The type of object expected on the Stream.
+ - Serializer to convert from the bytes stored in Pravega into the Event
+objects and a `ReaderConfig`.
+
+```Java
+EventStreamReader<String> reader = clientFactory.createReader("reader",
+                                                              readerGroup,
+                                                              new JavaSerializer<String>(),
+                                                              ReaderConfig.builder().build()))
+  ```                                        
+The name of the Reader can be any valid Pravega naming convention (numbers and letters). Note that the name of the Reader is namespaced within the Scope. `EventStreamWriter` and `EventStreamReader` uses Java generic types to allow a developer to specify a type safe Reader. In the sample application,
 we read Strings from the stream and use the standard Java String Serializer to
-convert the bytes read from the stream into String objects.  Finally, the
-ReaderConfig is created, but at the moment, there are no configuration items
-associated with a Reader, so the empty ReaderConfig is just a place holder as
+convert the bytes read from the stream into String objects. Finally, the
```

Comment:
I think that the last statement is not true. `ReaderConfig` has an attribute called `initialAllocationDelay`, so it is not empty. I would replace the last sentence by something like: `Finally, we use a ReaderConfig object with default values`.

Reply:
Accepted Suggestion